pyeudiw/jwt/utils.py

```diff
@@ -24,25 +24,32 @@ def decode_jwt_element(jwt: str, position: int) -> dict:
     :rtype: dict
     """
     if position < 0:
-        raise JWTInvalidElementPosition(f"Cannot accept negative position {position}")
+        raise JWTInvalidElementPosition(
+            f"Cannot accept negative position {position}"
+        )
 
     if position > 2:
         raise JWTInvalidElementPosition(
             f"Cannot accept position greater than 2 {position}"
         )
 
-    splitted_jwt = jwt.split(".")
-
-    if (len(splitted_jwt) - 1) < position:
-        raise JWTInvalidElementPosition(f"JWT has no element in position {position}")
-
     try:
+
         if isinstance(jwt, bytes):
             jwt = jwt.decode()
 
+        splitted_jwt = jwt.split(".")
+
+        if (len(splitted_jwt) - 1) < position:
+            raise JWTInvalidElementPosition(
+                f"JWT has no element in position {position}"
+            )
```

Comment:
please follow my previous comment

and also, what if position is none or 0?

this check should move out of the try statement as well